Storage/StorageManager.php

```diff
@@ -120,6 +120,9 @@ public function getFile(string $hash, ?string $context = null): string
 
     /**
      * @deprecated
+     * @param string $hash
+     * @param string|null $context
+     * @return string
      */
```

Comment:
As I wanted the @deprecated in the comment, I also added the rest of the commets to avoid PHP Storm warns